config/settings_schema.json

```diff
@@ -623,8 +623,8 @@
         "step": 5,
         "unit": "%",
         "label": "t:settings_schema.global.settings.opacity.label",
-        "default": 0
-      },      
+        "default": 10
```

Comment:
> but the fact that merchants could set it to zero might not be great and could highlight the fact that the borders of the accordions shouldn't be dependent from this setting maybe.

Good point! We should change the default just for that instance. Might have to come more in the future of how divider lines can be changed which we didn't consider as a priority for the latest changes we've made. Let's keep the divider line hard coded opacity like before and not change the default for that global settings as this would interfere with many other components. 🙏